static/test/javascripts-legacy/spec/commercial/hosted/about-popup.spec.js

```diff
@@ -19,7 +19,19 @@ define([
                 ]
             };
 
+
+
         beforeEach(function (done) {
+            var fakeSvg = {
+                markup: ''
+            };
+
+            injector.mock({
```

Comment:
Would it make sense to override the default `mock` [here](https://github.com/guardian/frontend/blob/master/static/test/javascripts-legacy/helpers/injector.js) for SVGs?